README.md

```diff
@@ -9,6 +9,7 @@ This repository list of proposals for the Strimzi project. A template for new pr
 
 |  #  | Title                                                                 |
 | :-: |:----------------------------------------------------------------------|
+| 56  | [Kafka Roller KRaft Support](./056-kafka-roller-kraft.md)
```

Comment:
You should rebase, use a different updated index and renaming the MD file as well.

Reply:
Yeah I will, I was holding off to see if there were any other comments as renaming the file sometimes makes it harder to see the previous discussions